Anoma/System.juvix

```diff
@@ -4,52 +4,71 @@ module Anoma.System;
 import Stdlib.Prelude open;
 
 syntax alias PublicKey := Nat;
+syntax alias PrivateKey := Nat;
 
 syntax alias Signature := Nat;
+-- The first part is the original message encoded as a `Nat`, the last part is the 64 bytes long signature.
```

Comment:
I think we should keep this interface abstracted from the particular cryptographic implementation that the Anoma Node (which could change). So we shouldn't document the `SignedMessage` structure. As far as the user is concerned, there's just some opaque `SignedMessage` which can be used as part of this interface.

Reply:
Fixed in f0263e199be6057937b0bc2f4b14400c78152b83